### PR TITLE
RMMIS-6139: Existence check for previous

### PR DESCRIPTION
**Pull request by traviscooperscdhhs**

### CODE REVIEW

File: src/ui-component-schema-utils.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ class SchemaUtils {
    */
   static getRootId(schema, componentId) {
     let meta = SchemaUtils.getMetaData(schema, componentId);
-    while(meta && !meta.parent) {
+    while(meta && !meta.parent && meta.previous) {
```
Comment (mallenscdhhs) [inline src/ui-component-schema-utils.js]: Why is this necessary? There is an existence check in `getMetaData`, right?

Comment (traviscooperscdhhs) [inline src/ui-component-schema-utils.js]: Results in infinite loop, if you're on the first top level item.  So the first top level item will never have a parent, and will not have a previous, so it never terminates the `while` loop.

Comment (mallenscdhhs) [inline src/ui-component-schema-utils.js]: (doh)!
